devel/hivex/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+name                hivex
+version             1.3.23
+revision            0
+categories          devel
+license             LGPL-2.1
+maintainers         {outlook.com:mohd.akram @mohd-akram} openmaintainer
+
+description         A self-contained library for reading and writing Windows \
+                    Registry \"hive\" binary files.
+
+long_description    {*}${description}
+
+homepage            https://github.com/libguestfs/hivex
+
+master_sites        https://download.libguestfs.org/hivex/
```

Comment:
can the default `masters_sites` from the PortGroup (i.e., CPAN) not be used here?

Reply:
This is not primarily a perl port. It is a C library that comes with perl bindings, and also a utility that uses perl (`hivexregedit`). I am not using the `perl5.setup` function which is normally used for perl ports, I'm only using the group to add perl variants (`perl5.create_variants` which only affects `depends_lib`). The port uses a standard `configure` and `make` so we do not really need or use anything from the PortGroup apart from the perl version.